scalableminds/webknossos
# Unify user vs owner field in annotation (it is now called owner)

### URL of deployed dev instance (used for testing):
- https://annotationowner.webknossos.xyz

### Steps to test:
- Create annotation, trace some, go to dashboard
- everything should still work as expected

------
- [ ] Adapted [wk-libs python client](https://github.com/scalableminds/webknossos-libs/tree/master/webknossos/webknossos/client) – API change is backwards compatible, but https://github.com/scalableminds/webknossos-libs/issues/743 should be done soon.
